docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -453,6 +453,12 @@ The `server` block configures the HTTP and gRPC server of the launched service(s
 [tls_min_version: <string> | default = ""]
 
 http_tls_config:
+  [cert: <string> | default = ""]
```

Comment:
These should be hidden in the config. I think you have to add `doc:hidden` to the respective fields in dskit.